zxcvbn-js.gemspec

```diff
@@ -13,10 +13,10 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.name          = "zxcvbn-js"
```

Comment:
hmm, does this need to change to `zxcvbnjs` too?

Reply:
We'll keep the gem name the same for now just to be compatible with the source. Maybe on day we will contribute back to them.`zxcvbnjs` is just namespacing the module so we can use 2 different ones at the same time.